js/siteorigin-panels.js

```diff
@@ -3469,13 +3469,23 @@ module.exports = Backbone.View.extend( {
         // Make sure we actually need to copy content.
         if( panelsOptions.copy_content && this.attachedToEditor && this.$el.is(':visible')) {
```

Comment:
Hrmm, that's whack. Yeah, we shouldn't rely on passing this check. We might fix that variable in the future and then break this particular content copy mechanism.